test/create.test.js

```diff
@@ -33,7 +33,10 @@ describe('create', function() {
 
     assert.ok(actual instanceof Circle);
     assert.ok(actual instanceof Shape);
-    assert.deepStrictEqual(Circle.prototype, expected);
+    assert.deepStrictEqual(Object.keys(Circle.prototype), Object.keys(expected));
+    Object.keys(expected).forEach(property => {
```

Comment:
arrow functions should wrap the param in parens (style consistency nit)

Reply:
done